### ISSUE TITLE
Updates on revenue share - creator side

### ISSUE THREAD

**Issue by dmtrjsg** [Joystream/atlas#4856]
## Context
Review of designs for the purpose of clarification of mechanics led to a number of key improvements to the rev share Creator side

Impacts issues:
- https://github.com/Joystream/atlas/issues/3844
- https://github.com/Joystream/atlas/issues/3845

## Scope
:one:
Current state
-- Current state -> Replace with timing variations of the "progress widget"

Progress widget (long bar)
- Change label from "progress" to "participation"; Add info from “Current state” widget. 
- Tokens distributed and USD denomination should be focus of the "progress widget". 

Channel balance
- Channel balance - keep as is or remove

:two:
- Add workflow that entices owner to stake tokens if they have any.
Add to the "start rev share" and add a modal in the end of revenue share linking to portfolio. 
🆕  we also need a button that "unlocks" tokens for creator after rev share is closed.

**Comment by KubaMikolajczyk** [comment]
@dmtrjsg 
All of the changes are implemented into the designs for the revenue share tab ✅

🎥 Video update: https://www.loom.com/share/8080fb0a497f4722bd3b4e895a27ccdc?sid=b31ed20f-1980-475a-8071-1528057a5489

📄 My token changelog: https://www.figma.com/file/OuywwClsegx2eotkii4ea4/My-token?type=design&node-id=925-139042&mode=design&t=ChLSAGDwPT6d4cqD-4

**Comment by dmtrjsg** [comment]
1. Change in the widgets looking good 👍 
2. In the confirmation popup for creator we invite them to share link to their token page. But afair the only place you can claim revenue share as holder is from your portfolio and there isn't anything on the token page that indicates the revenue share status? If so, the copy of the top toaster needs to be updated- link to token page removed.

<img width="425" alt="Screenshot 2023-10-02 at 14 16 15" src="https://github.com/Joystream/atlas/assets/93924525/72f075bf-c74a-4dad-95a5-e3fd5cabf72d">

3. Since we are adding staking controls for creators to the dashboard.. Creator would also need a way to "unlock" their own tokens from their dashboard after Rev share is closed.

**Comment by KubaMikolajczyk** [comment]
@dmtrjsg 
Thank you for the feedback!

2. As we spoke I would leave the link to the public token page in the snackbar for the user to copy, but...
I replaced the Revenue share secondary button in widget for a primary button.

📄 Channel changelog: https://www.figma.com/file/POw4j95jszfIP5bGfas2IC/Channel?type=design&node-id=2386-171791&mode=design&t=cHrrqhQVlR1D2Fy1-4

3. I added an option for a revenue share creator to unlock their tokens both on the dashboard and in an additional modal after closing it without unlocking tokens before:

📄 My token changelog: https://www.figma.com/file/OuywwClsegx2eotkii4ea4/My-token?type=design&node-id=925-139042&mode=design&t=Zt110HSslmX2qede-4
